### ISSUE TITLE
Asking to open the project folder even when open the directory using project explorer

### ISSUE THREAD

**Issue by NipunaMarcus** [ballerinalang/composer#4825]
**Description:**
when open the first folder of the package tree using the project explorer it asking to open the project folder 
![even opening the directory on explorer](https://user-images.githubusercontent.com/9109762/33945698-bffba26a-e045-11e7-9d75-6462df7437dc.png)

and then when press yes it will open another folder tree with one folder up the tree.
![2nd](https://user-images.githubusercontent.com/9109762/33945709-c671045a-e045-11e7-954e-c2dfa701bcab.png)

Is this the expected behavior or should it close the first tree and open the new tree with one folder up the tree or should it not ask to open the project directory as user have already open folders?
 
**Affected Product Version:**
Composer - 0.95.4    

**Steps to reproduce:**
1. Open 1st folder of your package folder tree using project explorer.
2. Double click the .bal file available at the end of the tree.
3. Click ok on the prompt.

**Related .bal file source**
```Ballerina
package balproject.pkg1;

import ballerina.net.http;

function main (string[] args) {
    endpoint<http:HttpClient> httpEndpoint {
        create http:HttpClient("https://postman-echo.com", {});
    }
     http:Request req = {};
    // Send a GET request to the specified endpoint
    http:Response resp = {};
    resp, _ = httpEndpoint.get("/get?test=123", req);
    
    string s = "mars";
    println(s);
    var df = s.contains("mar");
}
```

**Comment by kaviththiranga** [comment]
@NipunaMarcus 

The proper program dir root is the one suggested secondly (parent folder of balproject folder). 
Even-though you have opened the balproject folder, it is not the correct program dir.

Program dir is the directory from which you exceute the run command when you are running a bal file in package.

For example, when you need to run the package `balproject.pkg1`, You will first change working directory to the parent folder of balproject folder and execute `ballerina run balproject.pkg1`

**Comment by kaviththiranga** [comment]
Here, the `balproject` folder is not the program dir but a package folder inside the program directory. `ballerina-workspace` is the program directory for `balproject.pkg1` package.

**Comment by kaviththiranga** [comment]
On a side note, there was a bug where there are two folders getting opened when opening a sample from welcome page which has a package declaration. This is fixed with bdea60481abb4e6e6873bfe3707a627d6a07d9ae
